### PR TITLE
chore: added .sauced.yaml and newly generated CODEOWNERS file

### LINKED ISSUE #4037: Feature: Added .sauced.yml and update CODEOWNERS
As a developer of the OpenSauced app, I want to updated the CODEOWNERS file using `saucectl` so that the CODEOWNERS file can be more granular and up to date.

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Adds a .sauced.yaml to the project and a regenerated OWNERS file care of `pizza generate codeowners ./ -c ./.sauced.yaml`

## Related Tickets & Documents

Closes #4037

## Steps to QA

Check the PR to see if the CODEOWNERS file is value (GitHub lets you know)

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [x] Tier 4

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+next.config.js @brandonroberts @nickytonline
+npm-shrinkwrap.json @nickytonline @brandonroberts @zeucapua
+package.json @nickytonline @zeucapua @brandonroberts
+pages/404.tsx @zeucapua @brandonroberts @nickytonline
+pages/500.tsx @nickytonline
+pages/_app.tsx @nickytonline @zeucapua
+pages/_document.tsx @nickytonline @zeucapua
+pages/account-deleted.tsx @nickytonline
+pages/feed/index.tsx @brandonroberts @zeucapua @zeucapua
+pages/s/[org]/[repo]/contributors.tsx @zeucapua @brandonroberts @zeucapua
```
Comment (nickytonline) [inline CODEOWNERS:237]: @jpmcb, it looks like CODEOWNERS don't like some file base routing folders.

Comment (jpmcb) [reply CODEOWNERS:237]: Way ahead of ya:
- https://github.com/open-sauced/pizza-cli/issues/100
- https://github.com/open-sauced/pizza-cli/issues/101

Comment (zeucapua) [reply CODEOWNERS:237]: TIL CODEOWNERS needs escaped characters
image

**Review by BekahHW — APPROVED** [review]

**Review by jpmcb — APPROVED** [review]

**Pull request merged**